frontend/src/components/crawl-queue.ts

```diff
@@ -158,7 +199,7 @@ export class CrawlQueue extends LiteElement {
     const data: ResponseData = await this.apiFetch(
       `/archives/${this.archiveId}/crawls/${this.crawlId}/queue?offset=${
         (this.page - 1) * this.pageSize
-      }&count=${this.page * this.pageSize - 1}`,
+      }&count=${this.page * this.pageSize - 1}&regex=${this.regex}`,
```

Comment:
```suggestion
      }&count=${this.pageSize}&regex=${this.regex}`,
```

The `count` is actually the count now, instead of the ending offset